packages/mattermost-redux/src/actions/preferences.ts

```diff
@@ -149,6 +149,20 @@ export function setCustomStatusInitialisationState(initializationState: Record<s
     };
 }
 
+export function setPostPriorityInitialisationState(initializationState: Record<string, boolean>) {
```

Comment:
@ koox00 I think the web platform wants to avoid anything global if we know it's specific to some particular case. This action and selector defined below wouldn't be used again.
@hmhealey, What do you suggest should we move them closer to the post-priority tour tip component?

Reply:
Yeah, we could certainly do that. Seeing as these will only really be used by the PostPriority component, I think we could create an actions file and a selectors file in `components/post_priority` to store these